includes/class-wc-install.php

```diff
@@ -620,8 +620,11 @@ private static function create_options() {
 		add_option( 'woocommerce_demo_store', 'no', '', 'no' );
 
 		// Define initial tax classes.
-		WC_Tax::create_tax_class( __( 'Reduced rate', 'woocommerce' ) );
-		WC_Tax::create_tax_class( __( 'Zero rate', 'woocommerce' ) );
+		if ( self::is_new_install() ) {
+			// Moved behind only for new install.
```

Comment:
Can you please keep the original comment here on this line. Your added comment isn't needed as the conditional is clear enough to state what is going on.

Reply:
@roykho please review again